openwisp_radius/signals.py

```diff
@@ -0,0 +1,3 @@
+from django.dispatch import Signal
+
+radius_accounting_success = Signal(providing_args=['account_data'])
```

Comment:
Please chagne `account_data` to `accounting_data` here and anywhere else, because account and accounting are two different concepts which must not be confused.